Orange/widgets/data/owcontinuize.py

```diff
@@ -344,7 +346,9 @@ def normalize_by_span(var, data_or_dist, zero_based=True):
 def normalize_by_sd(var, data_or_dist):
     dist = _ensure_dist(var, data_or_dist)
     mean, sd = dist.mean(), dist.standard_deviation()
-    return normalized_var(var, mean, 1 / sd)
+    with np.errstate(divide='ignore'):
+        reciprocal_sd = np.divide(1, sd)
```

Comment:
Is there any reason to mute error instead of doing something more explicit, like `sd[sd < 1e-10] = 1` (as an approximation for `sd[sd==0] = 1`)?